roles/telemetry_verify_metrics/tasks/verify_rabbitmq_metrics.yml

```diff
@@ -0,0 +1,55 @@
+- name: Verify rabbitmq scrapeconfig exists
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_cr_test_id: RHOSO-1223
+    common_cr_list:
+      - kind: scrapeconfigs.monitoring.rhobs
+        name: telemetry-rabbitmq
+
+- name: Verify rabbitmq pods are running
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_pod_test_id: RHOSO-1241
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack
+    common_pod_list:
+      - rabbitmq-server-0
+      - rabbitmq-cell1-server-0
+
+- name: Check rabbitmq metric endpoint
+  ansible.builtin.shell: |
+    oc rsh openstackclient curl https://rabbitmq.openstack.svc:15691/metrics
+  register: result
+  changed_when: false
+  failed_when: result.rc >= 1
+
+- name: Check rabbitmq-cell1 metric endpoint
+  ansible.builtin.shell: |
+    oc rsh openstackclient curl https://rabbitmq-cell1.openstack.svc:15691/metrics
```

Comment:
Should this URL be available from outside the openstackclient pod?
Could you get the URL/IP address and make it more generic?
Is the route exposed? Or is the service URL available? (It's fine if it isn't, but if it is, we should try accessing it without the openstackclient pod)

Reply:
We don't have any route, only a service. Regarding making it more generic, I guess I could get all the rabbitmq CRs and from the name of each I could construct the URL.